rules/no-statement-after-end.js

```diff
@@ -51,7 +51,7 @@ const create = context => {
 			.filter(info => info.ended);
 
 		// If this segment or any previous segment is already ended, further statements are not allowed, report as an error.
-		if (ended.length) {
+		if (ended.length !== 0) {
```

Comment:
Sure, fine by me.

If this nitpick happens a lot for you, then it could probably deserve a linting rule, or options for the explicit-length-check rule. We could have another linting rule for comparing the result of `.indexOf`.